client/rhel/mgr-daemon/mgr-daemon.changes

```diff
@@ -1,3 +1,5 @@
+- Fix wrong bugzilla entry at changelog
+
```

Comment:
I think, without date and author above, this would be invalid. Will that be added later?
Do we really need a changes entry about fixing a changes entry? IMHO, customer-visible changes should be the focus.

Reply:
Well, this is correct.

Our tooling create the header automatically when we tag. To be able to collect multiple entires
we aggreed on just writing the changelog line.

We cannot submit without tagging and tagging always produce a header.
A Header without a changelog entry is not allowed, so we need to add one.